python/pylibraft/pylibraft/common/mdspan.pxd

```diff
@@ -37,3 +38,6 @@ cdef device_matrix_view[int8_t, int64_t, row_major] get_dmv_int8(
 
 cdef device_matrix_view[int64_t, int64_t, row_major] get_dmv_int64(
     array, check_shape) except *
+
+cdef optional[device_matrix_view[int64_t, int64_t, row_major]] create_optional(
```

Comment:
I don't know if cython will allow automatic overloads of this based on type, will it? We might need to rename this something like `make_optional_view_int64` or something.